aws/default_retryer.go

```diff
@@ -21,6 +21,7 @@ import (
 //    func (d retryer) MaxRetries() int { return 100 }
 type DefaultRetryer struct {
 	NumMaxRetries int
+	MinTime       int64
```

Comment:
Probably want to include a member for min throttle time as well, e.g. (`minThrottleTime`)  Since this is separate value than non-throttle retry delay minimum.

In addition need to document what the default behavior is if the members are zero.

This is also a good opportunity to consider changing the minimum times from a `int64` to `time.Duration`  time.Duration is much nicer to use than an integer representing seconds.

```go
type DefaultRetryer struct {
    NumMaxRetries        int
    MinimumRetryDelay    time.Duration
    MinimumThrottleDelay time.Duration
}
```